reo/validators.py

```diff
@@ -246,18 +246,18 @@ def validDependencies(self, name):
 
         return valid
 
-    def validCompleteHours(self, schedule_name,expected_counts):
+    def validCompleteHours(self, schedule_name, expected_counts):
```

Comment:
could you please add a description of the input params to the doc string for this method? It makes it much easier to understand what it is doing.

Reply:
@t-kwasnik FYI doc strings have to use triple double quotes to be accessible via built-in `help` https://www.python.org/dev/peps/pep-0257/